lib/ouija_tests.ml

```diff
@@ -1,74 +1,96 @@
-open OUnit2
+open Core.Std
+open Async.Std
+open Sentinel.Std
 
-let rest _ =
+let check exp op = fun _ -> return (exp = op ())
```

Comment:
Ok, gotcha. I'll get this fixed together with the CI job so that this PR has proper build feedback